Catalog section improvements (#2107)
SkylineCommunications/dataminer-docs: user-guide/Cloud_Platform/Catalog/About_the_Catalog_module.md

# About the Catalog module

> As part of dataminer.services, the Catalog module allows Skyline customers and DataMiner Strategic Partners to browse available DataMiner connectors, install packages, Visio drawings, Automation scripts, dashboards, etc. and deploy them to their DataMiner System.

Reviewer: ```suggestion
As part of dataminer.services, the Catalog module allows Skyline customers and DataMiner Strategic Partners to browse and deploy available DataMiner connectors, packages, Visio drawings, Automation scripts, dashboards, and more.
```
It doesn't have to be worded exactly like this, but the original sentence is a bit too complicated, I think.
